### DISCUSSION TITLE
ns-warning-low

### DISCUSSION THREAD

**Discussion by djhurlburt** [NightscoutFoundation/xDrip#2640] [Q&A]
I couldn't find a place on github to post this question; I did post it on the Facebook site (CGM in the Cloud) but no response.  I have nightscout running and figured out how to get IFTTT to trigger stuff through my Alexa device.  What I'm finding though is that (a) it is using 'predicted' BG to send a maker trigger, despite having "ALARM_TYPE" set to "simple"; and (b) it seems a maker trigger is sent regardless of whether the data is going up or down across a threshold.  In other words, when she goes below "BG_TARGET_BOTTOM" the maker trigger is sent, but then if she treats and goes above "BG_TARGET_BOTTOM", another trigger is sent.  I only care if she is going below the "BG_TARGET_BOTTOM", but not sure if there is a setting that will enable only going below the value to send a maker trigger?

If there is a better forum for this question, please help: I was unable to find it.  

Thanks

**Comment by Navid200** [comment]
xDrip itself has smart alert, which only triggers an alert if glucose is moving in the wrong direction.  Meaning you get a high alert only if glucose is going up, and you get a low alert if glucose is going down.

But, I don't know if there is such a setting in Nightscout to apply a similar logic for the Nightscout alerts.

**Reply by djhurlburt** [reply]
Thanks Navid; can you help me navigate to where I would ask the NS folks; I honestly am confused about the organizational structure of the NS github "sites".

**Reply by Navid200** [reply]
I will tag Patrick in your post in the facebook group.  That group is where Nightscout should be supported.
